movement/io/load_bboxes.py

```diff
@@ -0,0 +1,644 @@
+        See .
+    fps : float, optional
+        The video sampling rate. If None (default), the ``time`` coordinates
+        of the resulting ``movement`` dataset will be in frame numbers. If
+        ``fps`` is provided, the ``time`` coordinates  will be in seconds. If
+        the ``time`` coordinates are in seconds, they will indicate the
+        elapsed time from the capture of the first frame (assumed to be frame
+        0).
+    use_frame_numbers_from_file : bool, optional
+        If True, the frame numbers in the resulting dataset are
+        the same as the ones specified for each tracked bounding box in the
+        input file. This may be useful if the bounding boxes are tracked for a
+        subset of frames in a video, but you want to maintain the start of the
+        full video as the time origin. If False (default), the frame numbers
+        in the VIA tracks .csv file are instead mapped to a 0-based sequence of
+        consecutive integers.
+
+    Returns
+    -------
+    xarray.Dataset
+        ``movement`` dataset containing the position, shape, and confidence
+        scores of the tracked bounding boxes, and any associated metadata.
+
+    See Also
+    --------
+    movement.io.load_bboxes.from_via_tracks_file
+
+    References
+    ----------
+    .. [1] https://www.robots.ox.ac.uk/~vgg/software/via/
+
+
+    Examples
+    --------
+    Create a dataset from the VIA tracks .csv file at "path/to/file.csv", with
+    the time coordinates in frames, and setting the first tracked frame in the
+    file as frame 0.
+
+    >>> from movement.io import load_bboxes
+    >>> ds = load_bboxes.from_file(
+    ...     "path/to/file.csv",
+    ...     source_software="VIA-tracks",
+    ... )
+
+    Create a dataset from the VIA tracks .csv file at "path/to/file.csv", with
+    the time coordinates in seconds, and assuming t = 0 seconds corresponds to
+    the first tracked frame in the file.
+
+    >>> from movement.io import load_bboxes
+    >>> ds = load_bboxes.from_file(
+    ...     "path/to/file.csv",
+    ...     source_software="VIA-tracks",
+    ...     fps=30,
+    ... )
+
+    Create a dataset from the VIA tracks .csv file at "path/to/file.csv", with
+    the time coordinates in frames, and using the same frame numbers as
+    in the VIA tracks .csv file.
+
+    >>> from movement.io import load_bboxes
+    >>> ds = load_bboxes.from_file(
+    ...     "path/to/file.csv",
+    ...     source_software="VIA-tracks",
+    ...     use_frame_numbers_from_file=True.
+    ... )
+
+    Create a dataset from the VIA tracks .csv file at "path/to/file.csv", with
+    the time coordinates in seconds, and assuming t = 0 seconds corresponds to
+    the 0th frame in the full video.
+
+    >>> from movement.io import load_bboxes
+    >>> ds = load_bboxes.from_file(
+    ...     "path/to/file.csv",
+    ...     source_software="VIA-tracks",
+    ...     fps=30,
+    ...     use_frame_numbers_from_file=True,
+    ... )
```

Comment:
Because most of these examples are specific to the VIA-tracks format, maybe it's better to move them to the docstring of the `from_via_tracks_file`? Instead you can use a generic simple example here (basically swap the examples between the two functions). 

Since you alread have a "See also" section redirecting people to the `from_via_tracks_file` docstring, we can point people there for detailed format-specific examples/instructions.

Since this is currently the only bbox format we support, I don't feel very strongly about it, but may become relevant in the future.

Reply:
yes, nice idea ✨